modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,259 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+ifeval::["{context}" == "nodes-scheduler-pod-affinity"]
+:pod:
+endif::[]
+ifeval::["{context}" == "nodes-scheduler-node-affinity"]
+:node:
+endif::[]
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-overriding-operator-pod-affinity_{context}"]
+
+ifdef::olm[]
+= Controlling where an Operator is installed
+endif::olm[]
+
+ifdef::pod[]
+= Using pod anti-affinity to control where an Operator is installed
+endif::pod[]
+
+ifdef::node[]
+= Using node affinity to control where an Operator is installed
+endif::node[]
+
+By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.
+
+For example, some Operators require being scheduled on a node with a particular architecture, such as linux/amd64, linux/arm64, or Windows.
+
+Or, you might want Operators that work closely together to be scheduled on the same host, or on hosts located on the same rack. Alternatively, you might want your Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues.
```

Comment:
Based on the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-conjunctions), I think you don't need the comma after _Or_. WDYT?
```suggestion
Or you might want Operators that work closely together to be scheduled on the same host, or on hosts located on the same rack. Alternatively, you might want your Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues.
```